src/wallet/feebumper.cpp

```diff
@@ -47,7 +47,8 @@ static feebumper::Result PreconditionChecks(const CWallet& wallet, const CWallet
 
     // check that original tx consists entirely of our inputs
     // if not, we can't bump the fee, because the wallet has no way of knowing the value of the other inputs (thus the fee)
-    if (!wallet.IsAllFromMe(*wtx.tx, ISMINE_SPENDABLE)) {
+    isminefilter filter = wallet.IsWalletFlagSet(WALLET_FLAG_DISABLE_PRIVATE_KEYS) ? ISMINE_WATCH_ONLY : ISMINE_SPENDABLE;
```

Comment:
Could you use ISMINE_ALL to avoid conflicts with future ScriptPubKeyMan?

Reply:
ok, we should probably sync this behavior for https://github.com/bitcoin/bitcoin/pull/16944/files#diff-b2bb174788c7409b671c46ccc86034bdR2173 as well

Reply:
I'm checking for legacy spkm instead.